src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -47,13 +51,22 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the person list with {@code persons}.
+     * {@code persons} must not contain duplicate persons.
+     */
+    public void setTags(List<Tag> tags) {
+        this.tags.setTags(tags);
```

Comment:
![image](https://user-images.githubusercontent.com/78332456/193867583-5f32a3f1-3f9b-4f02-a384-ac4b5c6e39f9.png)

I am not sure if we should follow this or the current AB3

Reply:
I think we can follow AB3 for now.